agent/containers/images/Makefile

```diff
@@ -1,31 +1,6 @@
-# Base Makefile for building all images and tagging them
-
-# List of external Make targets:
-#
-#   These act on all default platforms' containers:
-#
-#     all (default):  make all images for all default platforms
-#     tds:  make Tool Data Sink images for all default platforms
-#     tm:  make Tool Meister images for all default platforms
-#     tag-<TYPE> (e.g., "tag-latest"):  apply specified tag to all images for all default platforms
-#     push:  push images for all default platforms with "<git commit hash>" and "v<full RPM version>" tags
-#     push-<TYPE> (e.g., "push-latest"):  push images for all default platforms with specified tag
-#
-#   These act on the indicated distribution's containers:
-#
-#     <DISTRO> (e.g., "fedora-34"):  make all images for the requested platform
-#     <DISTRO>-tds (e.g., "fedora-34-tds"):  make the TDS image for the requested platform
-#     <DISTRO>-tm (e.g., "fedora-34-tm"):  make the TM image for the requested platform
-#     <DISTRO>-tag-<TYPE> (e.g., "fedora-34-tag-alpha"):  apply the specified tag to the distro containers
-#     <DISTRO>-push (e.g., "fedora-34-push"):  push the specified containers
-#     <DISTRO>-push-<TYPE> (e.g., "fedora-34-push-alpha"):  push the specified containers
-#
-#   Utility targets:
+# Base Makefile for building images and tagging them.
 #
-#     clean:  remove build artifacts
-#     pkgmgr-clean:  clear the local package manager cache
-#     all-tags:  build all default distro "-tags.lis" files and verify that they are consistent.
-#     all-dockerfiles:  build all default distro ".repo" and ".Dockerfile" files
+# See the README.md file for a description of the interface.
```

Comment:
It doesn't matter to me.  If @ndokos likes it the other way, I'll drop the patch sets from this PR.

Reply:
Ideally, what I would like to have is `make help` which prints out the most common set of targets  (or examples of such)  in less than a "page" and refers to wherever the complete documentation can be found (whether as a block comment in the Makefile or in the README does not make much difference). SInce I knew that the README existed, that's where I went to find what I needed to find - I did not think to look at the block comment until @webbnh pointed it out.

Reply:
Well, since the default `all` target just prints that you shouldn't do that, maybe it should depend on a `help` target instead which prints an extended (but suitably brief) explanation. In fact, ideally this might be a `MAKE.md` that could stand along from the overall `README.md` (which could link to `MAKE.md`). How easy is it to format markdown on the command line? 🤔

Reply:
I think having a `help` target would be lovely, particularly if it were "automagic" (e.g., generated by introspection, or something).  However, the notion of having to keep _three_ bits of text up to date (the help text, the "module" header, and the external READ-ME file), is so unsatisfying.  😝  That means, at best, the `help` target would probably just refer the user to the `README.md` file, but that's problematic -- how is the user supposed to format it for human consumption, and what happens if the file is missing?  😞